sensum-pwa/src/ui/elements/Navbar.res

```diff
@@ -0,0 +1,19 @@
+@react.component
+let make = (~leftComponent: option<React.element>=?) => {
+  switch leftComponent {
+  | None => 
+    <div
+      className="fixed w-full bg-purple-900 border-purple-600 border-2">
+      <h1 className="pl-2 text-4xl text-purple-50">
+        {"sensum"->React.string}
+      </h1>
+    </div>
```

Comment:
What do you think about using `<header>` instead of `<div>`>?

And the border-2 is generating some UI inconsistencies, you can see it in the left border, so putting `border-b-2` resolves the problem.

![imagen](https://user-images.githubusercontent.com/39845488/163733984-6640aee7-4558-49eb-8925-4f18b453ce2d.png)

```suggestion
    <header
      className="fixed w-full bg-purple-900 border-purple-600 border-b-2">
      <h1 className="pl-2 text-4xl text-purple-50">
        {"sensum"->React.string}
      </h1>
    </div>
```